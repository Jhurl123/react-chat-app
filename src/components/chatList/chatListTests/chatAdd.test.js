import React from 'react';
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import ChatAdd from '../chatAdd'

describe('Chat Add Tests', () => {

  it('renders the UI without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ChatAdd />, div);
    ReactDOM.unmountComponentAtNode(div);
  })
  
  it('renders the ui as expected', () => {
    const tree = renderer
    .create(<ChatAdd />)
    .toJSON();
    expect(tree).toMatchSnapshot();  
  })

})