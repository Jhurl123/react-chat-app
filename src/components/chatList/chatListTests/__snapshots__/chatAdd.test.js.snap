// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chat Add Tests renders the ui as expected 1`] = `
<button
  aria-label="Open a new Conversation"
  className="makeStyles-addContainer-13"
  onClick={[Function]}
>
  <svg
    aria-hidden={true}
    className="MuiSvgIcon-root makeStyles-addIcon-14 MuiSvgIcon-fontSizeLarge"
    focusable="false"
    viewBox="0 0 24 24"
  >
    <path
      d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
    />
  </svg>
  <span
    className="makeStyles-convPrompt-15"
  >
    Start Conversation
  </span>
</button>
`;
